db/transaction.go

```diff
@@ -169,7 +169,7 @@ func (tx *Transaction) Commit(l zl.Logger) (ref types.Ref, err error) {
 	tx.closed = true
 
 	if !tx.wrote {
-		// No need to do anything.
+		ref = tx.basis.Ref()
```

Comment:
[taste] I find this style of return pretty scary. I use named return values for documentation purposes, and also write to them as locals, but try to remember to always return explicitly, so in this case:

```
return tx.basis.Ref(), nil
```

Reply:
Removed the named return values in this case